README.md

```diff
@@ -54,8 +54,50 @@ the CNFs produced can be shown UNSAT instantly.
 
 For `k = 6`, `n = 30`, the CNF takes on the order of 17,000 CPU hours to show UNSAT when parallelized using cube&conquer.
 
-The script in `utils/6hole-cube.c` outputs the list of cubes we used to solve `k = 6`, `n = 30`.
-We will add scripts soon for reproducing the entire parallel computation.
+# Cube and Conquer verification
+
+In order to verify that the resulting formula $F$ is indeed UNSAT, two things ought to be checked:
+
+ 1) **(Tautology proof)** The formula $F$ is UNSAT iff $F \land C_i$ is UNSAT for every $i$.
+ 2) **(UNSAT proof)** For each cube $C_i$, the formula $F \land C_i$ is UNSAT.
+
+The **tautology proof** is implied by checking that $F \land (\bigwedge_i \neg C_i)$ is UNSAT, as per Section 7.3 of [Heule and Scheucher](https://arxiv.org/abs/2403.00737).
+
+As a first step, we will generate the main CNF by running:
+```
+lake exe encode hole 6 30 cube_checking/vars.out > cube_checking/6-30.cnf
```

Comment:
At least on my computer this needs to be run within the `Lean/` folder.
```
cd Lean
lake exe encode hole 6 30 ../cube_checking/vars.out > ../cube_checking/6-30.cnf
```

Reply:
good catch, updated!

Reply:
This comment change should probably be made to the "Generating CNFs" section in the README as well. I'll add a PR for this.

Reply:
Done!